src/pages/ShopDetail/ShopDetail.js

```diff
@@ -1,10 +1,68 @@
 import React from 'react';
-
+import './ShopDetail.scss';
+import RestaurantReview from './RestaurantReview/RestaurantReview';
+import RestaurantInfo from './RestaurantInfo/RestaurantInfo';
 class ShopDetail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      wanted: false,
+    };
+  }
+  goToWrtingPage = e => {
+    this.props.history.push('/ShopDetail-reveiwWritingPage');
+  };
+
+  wanted = () => {
+    this.setState({
+      wanted: !this.state.wanted,
+    });
+  };
   render() {
+    const { wanted } = this.state;
     return (
       <div className="ShopDetail">
-        <div>ShopDetail 입니다</div>
+        <div>
+          <aside className="foodimg">
+            <img alt="스시작이미지" src="/images/shopDetail/1.jpeg" />
+            <img alt="스시작이미지" src="/images/shopDetail/2.jpeg" />
+            <img alt="스시작이미지" src="/images/shopDetail/3.jpeg" />
+            <img alt="스시작이미지" src="/images/shopDetail/4.jpeg" />
+          </aside>
+        </div>
+        <div className="inner">
+          <header className="restaurantHeader">
+            <div className="restaurantTitle">
+              <span>스시작</span>
+              <span className="grade">4.7</span>
+            </div>
+            <div className="restaurantTitleButton">
+              <button
+                className="reveiwAndLikeIcon write"
+                onClick={this.goToWrtingPage}
+              >
+                <i className="far fa-edit "></i>
+                리뷰쓰기
+              </button>
+              <button
+                className={
+                  wanted ? ' reveiwAndLikeIcon wanted' : 'reveiwAndLikeIcon'
+                }
```

Comment:
- 달라지는 부분에 대해서만 삼항 연산자를 적용해주세요!
```suggestion
                className={`reviewAndLikeIcon ${wanted ? 'wanted' : ''}`}
```